.github/workflows/auto-update.yml

```diff
@@ -1,12 +1,13 @@
 name: Auto-update
 on:
+  pull_request:
```

Comment:
We don't want to run this action on every PR

Reply:
fair we need to match it with the branch the action created

Reply:
If you name your branch any of these it will run this job on push
```yaml      
- "ulrikstrid/auto-update-pr"
- "anmonteiro/auto-update-pr"
```